usecase/projection/base.go

```diff
@@ -68,3 +98,31 @@ func (base *Base) Id() string {
 func (base *Base) Config() interface{} {
 	return base.config
 }
+
+func setupDefaultGithubMigration(
+	projectionId string,
+	migrationHelper *github_mh.GithubMigrationHelper,
+) {
+	projectionSnakeId := strcase.ToSnake(projectionId)
+
+	if migrationHelper.MaybeSourceURL == nil {
+		migrationDirectory := fmt.Sprintf(github_mh.MIGRATION_DIRECTORY_FORMAT, projectionSnakeId)
```

Comment:
Can we just panic if SourceURL is not valid? Looks like there is no generic way to build a path for every projection/future entity

Reply:
Will move the execute `MigrationHelper.Migrate()` logic to projection's `OnInit()`.